service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CoreCaseDataStoreLoader.java

```diff
@@ -13,10 +20,21 @@
 
 public class CoreCaseDataStoreLoader {
 
-    private static final ObjectMapper mapper = new ObjectMapper();
+    private static final ObjectMapper mapper = JsonMapper.builder()
+        .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+        .addModule(new ParameterNamesModule())
+        .addModule(new Jdk8Module())
+        .addModule(new JavaTimeModule())
+        .build();
+
+    private static final CaseConverter caseConverter = new CaseConverter(mapper);
 
     private CoreCaseDataStoreLoader() {
-        // NO-OP
+        new ObjectMapper();        // NO-OP
```

Comment:
Really, does sonar complains? Even in tests? ;o 

What if there is no constructor, it's test class so I wouldn't worry..